css/responsive/tablet.styl

```diff
@@ -41,11 +41,8 @@ for col in 1..$cols
 // ARCHIVE EDITORIAL
 
 #posts-archive-editorial
-  height: 100%
-  min-height: 600px
   .col
-    height: 100%
-    min-height: 600px
+    min-height: 620px
```

Comment:
cause i would think the minheight would be less than that. at least to fit the footer on my screen without having it fixed